src/main/scala/gestalt/dotty/Expander.scala

```diff
@@ -67,7 +76,8 @@ object Expander {
   /** Expand def macros */
   def expandDefMacro(tree: tpd.Tree)(implicit ctx: Context): untpd.Tree = tree match {
     case ExtractApply(Select(obj, method), targs, argss) =>
-      val className = obj.symbol.info.classSymbol.fullName + "$inline$"
+      val objType = obj.symbol.info.resultType
+      val className = javaClassName(objType.classSymbol) + "$inline$"
```

Comment:
Here seems we can use `obj.tpe.widen.classSymbol`. We need to change `ExtractApply` to work on typed trees. BTW, could you please help refactor `obj` to `prefix`? Thanks.